docs/ruleset_functions/zone_meets_G3_1_1e.md

```diff
@@ -0,0 +1,60 @@
+- **B-RMR**
+- **zone**
+
+**Returns:**  
+- **result**: a string - either "E" or "No"
+ 
+**Function Call:**
+- **is_a_vestibule**
+
+## Logic:
+- create list of eligible space types: `eligible_space_types = ["STORAGE_ROOM_HOSPITAL", "STORAGE_ROOM_SMALL", "STORAGE_ROOM_LARGE", "WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS", "WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS", "STAIRWELL", "ELECTRICAL_MECHANICAL_ROOM", "RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED", "RESTROOM_ALL_OTHERS"]
+- set the result variable to "No" - only a positive test can give it a different value: `result = "No"`
+- set eligibility boolean to False: `eligible = TRUE`
+- For each space in zone: `for space_p in zone.Spaces:`
+	- check if the space has an eligible lighting type: `if space_p.lighting_space_type not in eligible_space_types:`
+		- set eligibility to False - any non-compliant space type will result in a non-eligible zone: `eligible = FALSE`
+
+- if the zone is not eligible, check if it is a vestibule: `if eligible == FALSE:`
+	- set variable is_a_vestibule = is_zone_a_vestibule: `is_a_vestibule = is_zone_a_vestibule(zone, B-RMR)`
+	- if the result is not equal to no, then it is eligible to be a vestibule: `if is_a_vestibule != "NO":`
+		- set eligible to true: `eligible = TRUE`
+
+- if the zone is still eligible: `if eligible == TRUE:`
+	- check if the proposed has a heating-only system
+	- get the building_segment_p the zone is in:
+	`for building_segment_p in P-RMR.building.building_segments:`
+		- check if the zone is in the building_segment:
+		`if zone in building_segment.zones:`
+			- break - this means the variable "building_segment" is the building_segment that contains the zone:
+			`break`
+	- loop through each heating system: `for system_p in building_segment.heating_ventilation_air_conditioning_systems:`
+		- look for a cooling system: `if system_p.cooling_system != "Null":`
+			- check to see if the cooling_system_options equals "None": `if system_p.cooling_system.cooling_system_options == "None":`
+				- the zone is eligible
+			- otherwise: `else:`
+				- there is a cooling system and the zone is not eligible: `eligible = FALSE`
+
+- if the zone is still eligible: `if eligible == TRUE:`
+	- set result to "E": `result = "E"`
+	- however, if there is transfer air into the space from an air conditioned zone, it is not eligible, so look for transfer air into the space: `if zone.transfer_airflow_rate > 0:`
+		- there is no way of knowing where the transfer air is coming from, so for now will give the result the tag "E_T" to signal that there is transfer air.  When the rule is written, "E_T" from this function will trigger an "UNDETERMINED" result and indicate manual checking for transfer air from air-conditioned space: `result = "E_T"
+
+
+**Returns** `result`
+
+
+**Notes/Questions:**  
+1. the "is_zone_a_vestibule" function gives only "NO" and indeterminite answers.  Is there a way to alert to confirm that the zone is a vestibule?

```

Comment:
I suggest setting it to `False` first instead of a string. A boolean is always a preferred way to string :)

Reply:
I realize this issue when reviewing `is_zone_a_vestibule` function. We can discuss it in the coming call. Thanks!